Update service-bus-nodejs-how-to-use-topics-subscriptions.md

JasonJJFS opened the pull request:
Missing closing parenthesis on   second example for "serviceBusService.receiveSubscriptionMessage"

azurecla commented:
Hi __@DavidHoerster__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

JasonJJFS committed:
Update service-bus-nodejs-how-to-use-topics-subscriptions.md

Missing closing parenthesis on   second example for "serviceBusService.receiveSubscriptionMessage"
